### ISSUE TITLE
Force/Changing Default Framerate to correct FPS of Exported AV1 video?

### ISSUE THREAD

**Issue by Corruptinator** [xiph/rav1e#1667]
I was testing compiling an image sequence to the av1 video using **ffmpeg** and **rav1e** and realized that the video exported from the encoding process says that the framerate is at the defaulte 25fps:

```
ffmpeg -i Frame_%04d.exr -strict -1 -pix_fmt yuv420p10 -r 24 -f yuv4mpegpipe - | rav1e - --speed 8 --primaries "BT2020" --transfer "PerceptualQuantizer" --matrix "BT2020NonConstantLuminance" --range "Full" --output "New_HEVC_Test.ivf"
```
The way the framerate would be specified would be either by  "-**framerate <fps>**", or "**-r <fps>**".

![image](https://user-images.githubusercontent.com/5027750/65000906-4c6ca180-d8a2-11e9-8ec9-743e0df0d142.png)

I thought the ffmpeg part with the framerate force isnt working, until I checked through rav1e's -help, -fullhelp, and other possible documentations, rav1e doesn't have an "fps" command argument.

Bottom line: is it possible for the rav1e encoder have a forced framerate/ fps adjust argument command so then it can correctly play through frames at the specified framerate? Thanks.

**Comment by tdaede** [comment]
Hmm, passing rate through with -r should actually work. This might be a bug in the encoder wrapper right now.
